docs/blockchain_tests/index.rst

```diff
@@ -0,0 +1,139 @@
+.. _blockchain_tests:
+
+################################################################################
+Blockchain Tests
+################################################################################
+
+Found in ``/BlockTests``, the blockchain tests aim is to test the basic verification of a blockchain.
+
+``/BlockTests`` - general blockchain tests. All blocks are built on network: **Frontier** 
+``/BlockTests/Homestead`` - homestead blockchain tests. All blocks are built on network: **Homestead** 
+``/BlockTests/TestNetwork`` - transition blockchain tests. All blocks before 5th are built on network: **Frontier**, then each block should correspond to Homestead rules.
+
+It is based around the notion of executing a list of single blocks, described by the ``blocks`` portion of the test. The first block is the modified genesis block as described by the ``genesisBlockHeader`` portion of the test. A set of pre-existing accounts are detailed in the ``pre`` portion and form the world state of the genesis block. 
+
+It is generally expected that the test implementer will read ``genesisBlockHeader`` and ``pre`` and build the corresponding blockchain in the client. Then the new blocks, described by its RLP found in the ``rlp`` object of the ``blocks`` (RLP of a complete block, not the block header only), is read. If the client concludes that the block is valid, it should execute the block and verify the parameters given in ``blockHeader`` (block header of the new block), ``transactions`` (transaction list) and ``uncleHeaders`` (list of uncle headers). If the client concludes that the block is invalid, it should verify that no ``blockHeader``, ``transactions`` or ``uncleHeaders`` object is present in the test. The client is expected to iterate through the list of blocks and ignore invalid blocks.
+
+Basic structure
+--------------------------------------------------------------------------------
+
+::
+
+	{
+	   "ValidBlocks": {
+		   "genesisBlockHeader": { ... },
+		   "pre": { ... },
+		   "blocks" : [
+			   {
+				   "chainname" : "A",
```

Comment:
optional field. we should leave it only in the test fillers. 
additional or optional fields in the final tests are confusing other teams